test/intl402/DurationFormat/prototype/resolvedOptions/return-keys-order-default.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-intl.durationformat.prototype.resolvedoptions
+description: order of property keys for the object returned by resolvedOptions()
+features: [Intl.DurationFormat]
+---*/
+
+assert.sameValue(
```

Comment:
`assert.compareArray` can also be used here.

Reply:
done